web/html/src/components/input/Text.tsx

```diff
@@ -34,6 +34,7 @@ export const Text = (props: Props) => {
             className={`form-control${inputClass ? ` ${inputClass}` : ""}`}
             type={type || "text"}
             name={props.name}
+            id={props.name}
```

Comment:
This isn't really safe and may lead to unexpected bugs. The problem here is that the name property could be anything and it isn't clear from the outside that it will also be used as an id. To exemplify, it would be perfectly valid to have a field with a name of say `network` but also have a DOM node with the same id somewhere else on the page, perhaps in a completely unrelated component. This would then cause all sorts of unexpected problems.  

The fix is simple, `id` should be a prop by itself and it should be explicitly set when needed.

Reply:
The `for` part is generated [here](https://github.com/uyuni-project/uyuni/blob/master/web/html/src/components/input/InputBase.tsx#L255) in the `InputBase` component and uses the field name as well. This is already done this way in other form components, just that this one was left aside for some reason.

If someone wants to fix this at the `InputBase` level and in all the forms using it why not... but I won't start such a task myself.

Reply:
Oh okay, that makes me sad. In that case this is okay, I'll create a ticket to clean this up in the future.

Reply:
Created https://github.com/SUSE/spacewalk/issues/14165 to track this.

Reply:
thanks for tracking this!